Sources/Files.swift

```diff
@@ -298,6 +298,24 @@ public class FileSystem {
         return try! Folder(path: ProcessInfo.processInfo.homeFolderPath, using: fileManager)
     }
     
+    /// A reference to the document folder used by this file system
+    public var documentFolder: Folder? {
+        guard let url = try? fileManager.url(for: .documentDirectory, in: .userDomainMask, appropriateFor: nil, create: false) else { return nil }
+        
+        do {
+            return try Folder(path: url.path, using: fileManager)
+        } catch {
+            return nil
+        }
+    }
+    
+    /// A reference to the library folder used by this file system
+    public var libraryFolder: Folder {
```

Comment:
I removed the test for this reference, because it was consistently available on iOS, macOS and tvOS. I've not tested the library on Linux, so I will go through it again.